support-frontend/assets/pages/[countryGroupId]/components/thankyou.tsx

```diff
@@ -150,14 +150,15 @@ export function ThankYouComponent({
 				: order.paymentMethod;
 
 		successfulContributionConversion(
-			payment.originalAmount,
+			payment.finalAmount, // This is the final amount after discounts
 			contributionType,
 			currencyKey,
 			paymentMethod,
+			productKey ?? 'Contribution', // One-off is labelled Contribution in Tag Manager
 		);
 		// track conversion with QM
 		sendEventContributionCheckoutConversion(
-			payment.originalAmount,
+			payment.originalAmount, // This is the amount before discounts
```

Comment:
Why do we send finalAmount to GTM but originalAmount to QM?

Reply:
I know, spoke with George about this and will moved QM over to `finalAmount` in a seperate QM PR so its clear.